frontend/playwright.config.ts

```diff
@@ -1,12 +0,0 @@
-import type { PlaywrightTestConfig } from '@playwright/test';
-
-const config: PlaywrightTestConfig = {
```

Comment:
We need this file for if we decide to do automated testing in the future.

Reply:
My bad, I didn't know that. I'll fix it